lib/memflash.rb

```diff
@@ -32,7 +32,7 @@ def memflash_key(hash_key)
     end
 
     def memflashed?(key, value)
-      /^Memflash-#{key}/.match? value
+      !!(value =~ /^Memflash-#{key}/)
```

Comment:
this is a mysterious way to do this really.

```
irb(main):015:0> /abcd/ =~ true
Traceback (most recent call last):
        2: from /Users/vanchi/.rbenv/versions/2.5.8/bin/irb:11:in `<main>'
        1: from (irb):15
TypeError (no implicit conversion of true into String)
irb(main):016:0> true =~ /abcd/
=> nil
```
Should we simply be more explicit? That is skip if value is not a string or a symbol ?